src/raft.c

```diff
@@ -599,6 +637,37 @@ static void clearClientSessions(RedisRaftCtx *rr)
     rr->client_session_dict = RedisModule_CreateDict(rr->ctx);
 }
 
+static void timeoutBlockedCommand(RedisRaftCtx *rr, raft_entry_t *entry)
+{
+    raft_index_t idx;
+    bool error; /* not used yet */
+
+    RedisModule_Assert(RaftRedisDeserializeTimeout(entry->data, entry->data_len, &idx, &error) != RR_OK);
```

Comment:
I'm confused by this ` != RR_OK`. Seems like we expect this to fail? 

Btw, could you move this function outside of assert. If something is in assert, it gives impression that it is just there to validate things.

Reply:
you're right.  ok.  I'll assert on the response.  that's probably in a few places.